docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+                }
+            }
+        },
+        {
+            "name": "ledgerlive.account.sign_transaction",
+            "summary": "Sign a transaction",
+            "description": "Sign a transaction",
+            "paramStructure": "by-name",
+            "params": [
+                {
+                    "name": "accountId",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "transaction",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/Transaction"
+                    }
+                }
+            ],
+            "result": {
+                "name": "signedTransaction",
+                "description": "The signed transaction buffer",
+                "schema": {
+                    "$ref": "#/components/schemas/ByteBuffer"
+                }
+            },
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Transactions"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionTransaction"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.account.broadcast",
+            "summary": "Use Ledger Live to broadcast a signed transaction",
+            "paramStructure": "by-name",
+            "params": [
+                {
+                    "name": "accountId",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "signedTransaction",
+                    "schema": {
+                        "$ref": "#/components/schemas/ByteBuffer"
+                    }
+                }
+            ],
+            "result": {
+                "$ref": "#/components/schemas/UnitSuccessResponse"
+            },
+            "tags": [
+                {
+                    "$ref": "#/components/tags/Transactions"
+                },
+                {
+                    "$ref": "#/components/tags/PermissionTransaction"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.account.sign_message",
+            "summary": "Sign an UTF-8 encoded message with a user private key.",
+            "description": "Sign an UTF-8 encoded message with a user private key.",
+            "paramStructure": "by-name",
+            "params": [
+                {
+                    "name": "account_id",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/AccountId"
+                    }
+                },
+                {
+                    "name": "path",
+                    "required": true,
```

Comment:
Why a path in addition to an account_id? Why not in sign_transaction?

Reply:
Because in sign transaction, you are actually signing the transaction N times for N UTXO, each UTXO have it's own private key, you can't mix. Where as for sign message, you  sign it for  a given address not account (for BTC at least)

The path is not useful for single private key currencies